test/e2e/storage/utils/create.go

```diff
@@ -62,6 +62,10 @@ func LoadFromManifests(files ...string) ([]interface{}, error) {
 		if err := runtime.DecodeInto(scheme.Codecs.UniversalDecoder(), data, &what); err != nil {
 			return fmt.Errorf("decode TypeMeta: %w", err)
 		}
+		// Ignore empty documents.
```

Comment:
When will we get empty documents?

Reply:
please see 
https://github.com/kubernetes/kubernetes/blob/11448f8d8cda4766508f9a1ffcae9408451fc556/test/e2e/testing-manifests/storage-csi/external-snapshotter/csi-snapshotter/rbac-csi-snapshotter.yaml#L1-L15